.gitignore

```diff
@@ -1 +1,3 @@
 target
+.idea
```

Comment:
This is specific to your system and I'd prefer this to be added to your global gitignore, not in the project